### PR TITLE
LPS-87417

### PR DESCRIPTION
**Pull request by petershin**

### CODE REVIEW

**Comment by petershin** [comment]
@lawrence-lee

**Comment by petershin** [comment]
ci:test:sf

**Comment by petershin** [comment]
ci:test:relevant

**Comment by petershin** [comment]
ci:test:gradle-plugins

File: modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/constants/BundleSupportConstants.java [hunk 1]
```diff
@@ -27,11 +27,11 @@
 	public static final String DEFAULT_BUNDLE_CACHE_DIR_NAME =
 		".liferay/bundles";
 
-	public static final String DEFAULT_BUNDLE_FORMAT = "zip";
+	public static final String DEFAULT_BUNDLE_FORMAT = "tar.gz";
```
Comment (gamerson) [inline modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/constants/BundleSupportConstants.java]: Will this cause a problem for previous releases?

**Comment by petershin** [comment]
Waiting for final ok from @lawrence-lee

**Comment by lawrence-lee** [comment]
@petershin can you send this back to me? I'll go ahead and work on it and test out @gamerson's question

**Comment by petershin** [comment]
Sent to Lawrence: https://github.com/lawrence-lee/liferay-portal/pull/99

**Pull request closed without merging**